src/algebraic_geometry/prime_spectrum.lean

```diff
@@ -66,6 +66,12 @@ abbreviation as_ideal (x : prime_spectrum R) : ideal R := x.val
 instance is_prime (x : prime_spectrum R) :
```

Comment:
I can't comment above, but please:
- add `@[nolint has_inhabited_instance]` to `prime_spectrum`, 
- update the docstring: the structure sheaf is now defined,
- fix the typo in the docstring: "the type of all prime ideal**s** of `R`."

Reply:
Done!